### PR TITLE
[#450] Replace urdu numerals to use standard numerals

### PR DESCRIPTION
**Pull request by tonythomas01**
Convention on urwiki is to use Latin numerals, which is not possible with their favoured (phonetic) IME.

https://github.com/wikimedia/jquery.ime/blob/HEAD/rules/ur/ur-phonetic.js is the definition with `[ '1', '۱' ]`rather than `[ '1', '1' ]`

(Reported in person by @hindustanilanguage.) in https://phabricator.wikimedia.org/T138399

### CODE REVIEW

**Comment by kartikm** [comment]
Patch looks good to me. It would be nice if you add 2 or 3 tests at, test/jquery.ime.test.fixtures.js Look at similar tests from other languages.

File: rules/ur/ur-phonetic.js [hunk 1]
```diff
@@ -12,20 +12,20 @@
 		version: '1.0',
 		patterns: [
 			[ '!', '!' ],
-			[ '1', '۱' ],
-			[ '2', '۲' ],
+			[ '1', '1' ],
```
Comment (santhoshtr) [inline rules/ur/ur-phonetic.js]: You can just remove the mapping for the numeral. [ '1', '1' ] has no effect(it means when you type 1, produce 1, which anyway is the default behavior

Comment (tonythomas01) [inline rules/ur/ur-phonetic.js]: Thats right. Dont know why I did that in the first place. Removing now

**Comment by tonythomas01** [comment]
I have pushed in a test too, which I am ashamed to admit that I couldn't test personally :(

**Pull request merged**